CMakeLists.txt

```diff
@@ -77,27 +79,39 @@ if( BUILD_APPLICATIONS )
   add_subdirectory( app )
 endif()
 
-export( TARGETS IntersonArrayCxx
-  FILE "${PROJECT_BINARY_DIR}/IntersonArraySDKCxxTargets.cmake" )
-export( PACKAGE IntersonArraySDKCxx )
+# Configure 'IntersonArraySDKCxxConfig.cmake' for a build tree
+set(INCLUDE_DIR_CONFIG ${INCLUDE_BUILD_DIR})
+set(intersonArraySDKCxx_config ${PROJECT_BINARY_DIR}/IntersonArraySDKCxxConfig.cmake)
+configure_package_config_file(
+    IntersonArraySDKCxxConfig.cmake.in
+    ${intersonArraySDKCxx_config}
+    INSTALL_DESTINATION ${PROJECT_BINARY_DIR}
+    PATH_VARS CONF_INCLUDE_DIRS
```

Comment:
It is defined line 39. It was set two times in the previous CMakeLists (l.37 and 90)